source/docs/index.md

```diff
@@ -1,102 +1,23 @@
 
 ## What is Embark?
 
-Embark is a fast, simple and powerful framework to develop and deploy Decentralized Applications (DApps). 
+Embark is a fast, easy to use and powerful developer environment to build and deploy decentralized applications, also known as "DApps". It integrates with [Ethereum](https://ethereum.org/) blockchains, decentralized storages like [IPFS](https://ipfs.io) and [Swarm](https://swarm-guide.readthedocs.io/en/latest/introduction.html), and decentralized communication platforms like [Whisper](https://github.com/ethereum/wiki/wiki/Whisper) and [Orbit](https://github.com/orbitdb/orbit).
 
-Embark currently integrates with EVM blockchains (Ethereum), Decentralized Storages (IPFS), and Decentralized communication platforms (Whisper and Orbit). Swarm is supported for deployment.
+Embark's goal is to make building decentralized applications as easy as possible, by providing all the tools needed and staying extensible at the same time.
 
-With Embark you can:
+Some of Embark's features, but not all of them, are:
 
-**Blockchain (Ethereum)**
-* Automatically deploy contracts and make them available in your JS code. Embark watches for changes, and if you update a contract, Embark will automatically redeploy the contracts (if needed) and the dapp.
-* Contracts are available in JS with Promises.
-* Do Test Driven Development with Contracts using Javascript.
-* Keep track of deployed contracts; deploy only when truly needed.
-* Manage different chains (e.g testnet, private net, livenet)
-* Easily manage complex systems of interdependent contracts.
+* **Automatic Smart Contract deployment** - Embark takes care of deploying your Smart Contracts as well as redeploying them as you make changes to your code.
+* **Client development** - Smart Contracts are not a must! Build your application with the framework of your choice right within Embark.
+* **Testing** - Test your applications and Smart Contracts through Web3 in JavaScript
+* **Decentralized app distribution** - Embark integrates with decentralized storages like IPFS and helps you distributing your app in the network.
+* **Peer-to-peer messaging** - Send and receive messages via communication protocols like Whisper and Orbit
 
-**Decentralized Storage (IPFS)**
-* Easily Store & Retrieve Data on the DApp through EmbarkJS. Including uploading and retrieving files.
-* Deploy the full application to IPFS or Swarm.
+There's so much more, but let's take it one step at a time. Make sure to check out our other guides in case you're looking for a particular feature.
 
+If you haven't yet, this is a good time to install Embark on your machine. Read on for more information!
```

Comment:
It might make sense to introduce another "Useful Resources" section here with links to gitter, github etc, just so we have quick access for the reader without her parsing the whole text above for links.